common/lib/xmodule/xmodule/course_module.py

```diff
@@ -202,6 +202,15 @@ class CourseFields(object):
         default=0,
         scope=Scope.settings,
     )
+    course_google_analytics_tag = String(
```

Comment:
I'd like to split this out into a Stanford-specific file/object,
and pull it in via mixin.